### DISCUSSION TITLE
Debugger statements in production

### DISCUSSION THREAD

**Discussion by ivan-kleshnin** [vercel/next.js#15815] [Help]
I have some errors that occur only in production builds so I need to debug them there. 
Can I prevent NextJS from removing `debugger` statements during the `next build` process?

**Comment by Oikio** [comment]
Can't help with this debugger statement itsulef, but you can connect debugger (e.g. Chrome's Node debugger) to your running app if you start it with `NODE_OPTIONS='--inspect'`

**Comment by crs1138** [comment]
@ivan-kleshnin did you manage to get the `debugger` statement into production? I'm facing the same problem and @Oikio solution does not really apply to my case.

**Reply by ivan-kleshnin** [reply]
@crs1138 I don't remember, sorry 😢
